src/genie/libs/parser/iosxe/show_gnxi.py

```diff
@@ -0,0 +1,243 @@
+            gnxi_state_detail_dict["settings"]["secure_trustpoint"] = str(groups["secure_trustpoint"])
+            gnxi_state_detail_dict["settings"]["secure_client_trustpoint"] = str(groups["secure_client_trustpoint"])
+            gnxi_state_detail_dict["settings"]["secure_password_authentication"] = enabled_disabled_to_bool(groups["secure_pass_auth"])
+
+            # gnmi broker information
+            gnxi_state_detail_dict["oper_state"]["admin_enabled"] = enabled_disabled_to_bool(groups["admin_state"])
+            gnxi_state_detail_dict["oper_state"]["oper_up"]       = up_down_to_bool(groups["oper_status"])
+            gnxi_state_detail_dict["oper_state"]["provisioned"]   = groups["bootstrapping_state"] == "Provisioned"
+
+            # grpc server information
+            gnxi_state_detail_dict["oper_state"]["grpc"]["admin_enabled"] = enabled_disabled_to_bool(groups["grpc_admin_state"])
+            gnxi_state_detail_dict["oper_state"]["grpc"]["oper_up"] = up_down_to_bool(groups["grpc_oper_status"])
+
+            # configuration interface information
+            gnxi_state_detail_dict["oper_state"]["config_svc"]["admin_enabled"] = enabled_disabled_to_bool(groups["config_svc_admin_state"])
+            gnxi_state_detail_dict["oper_state"]["config_svc"]["oper_up"] = up_down_to_bool(groups["config_svc_oper_status"])
+
+            # telemetry interface information
+            gnxi_state_detail_dict["oper_state"]["telemetry_svc"]["admin_enabled"] = enabled_disabled_to_bool(groups["telemetry_svc_admin_state"])
+            gnxi_state_detail_dict["oper_state"]["telemetry_svc"]["oper_up"] = up_down_to_bool(groups["telemetry_svc_oper_status"])
+
+            # certificate management interface information
+            gnxi_state_detail_dict["oper_state"]["cert_mgmt_svc"]["admin_enabled"] = enabled_disabled_to_bool(groups["cert_svc_admin_state"])
+            gnxi_state_detail_dict["oper_state"]["cert_mgmt_svc"]["oper_up"] = up_down_to_bool(groups["cert_svc_oper_status"])
+
+            # os image interface information
+            gnxi_state_detail_dict["oper_state"]["os_image_svc"]["admin_enabled"] = enabled_disabled_to_bool(groups["os_svc_admin_state"])
+            gnxi_state_detail_dict["oper_state"]["os_image_svc"]["oper_up"] = up_down_to_bool(groups["os_svc_oper_status"])
+            gnxi_state_detail_dict["oper_state"]["os_image_svc"]["supported"] = groups["os_svc_supported"] == "Supported"
+
+            # factory reset interface information
+            gnxi_state_detail_dict["oper_state"]["factory_reset_svc"]["admin_enabled"] = enabled_disabled_to_bool(groups["reset_svc_admin_state"])
+            gnxi_state_detail_dict["oper_state"]["factory_reset_svc"]["oper_up"] = up_down_to_bool(groups["reset_svc_oper_status"])
+            gnxi_state_detail_dict["oper_state"]["factory_reset_svc"]["supported"] = groups["reset_svc_supported"] == "Supported"
+
+        else:
+            gnxi_state_detail_dict = {}
```

Comment:
This will need to change along with your regex. Again, please look at existing parsers for examples